app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala

```diff
@@ -45,28 +44,30 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class StartController @Inject()(
-                                 bprService: BusinessPartnerRecordService,
-                                 val sessionStore: SessionStore,
-                                 val errorHandler: ErrorHandler,
-                                 cc: MessagesControllerComponents,
-                                 val authenticatedActionWithRetrievedData: AuthenticatedActionWithRetrievedData,
-                                 val sessionDataActionWithRetrievedData: SessionDataActionWithRetrievedData,
-                                 val authenticatedAction: AuthenticatedAction,
-                                 val sessionDataAction: SessionDataAction,
-                                 val config: Configuration,
-                                 weNeedMoreDetailsPage: views.html.we_need_more_details
+  bprService: BusinessPartnerRecordService,
+  val sessionStore: SessionStore,
+  val errorHandler: ErrorHandler,
+  cc: MessagesControllerComponents,
+  val authenticatedActionWithRetrievedData: AuthenticatedActionWithRetrievedData,
+  val sessionDataActionWithRetrievedData: SessionDataActionWithRetrievedData,
+  val authenticatedAction: AuthenticatedAction,
+  val sessionDataAction: SessionDataAction,
+  val config: Configuration,
+  weNeedMoreDetailsPage: views.html.we_need_more_details
 )(implicit viewConfig: ViewConfig, ec: ExecutionContext)
     extends FrontendController(cc)
     with WithAuthRetrievalsAndSessionDataAction
-      with WithAuthAndSessionDataAction
-      with Logging
+    with WithAuthAndSessionDataAction
+    with Logging
     with SessionUpdates
     with IvBehaviour {
 
   def start(): Action[AnyContent] = authenticatedActionWithRetrievedDataAndSessionData.async { implicit request =>
-
     (request.authenticatedRequest.userType, request.sessionData.flatMap(_.journeyStatus)) match {
 
+      case (UserType.Subscribed(cgtReference, gGCredId), None) =>
```

Comment:
would we not need a case for `(_, _: JourneyStatus.Subscribed)` as well?

Reply:
would we not need to write the a `Subscribed` journey status here in session to allow for the `HomeController` to pick it up and not redirect back to start?